blocks/right-rail-block/index.story.jsx

```diff
@@ -45,3 +45,14 @@ export const basic = () => (
 		<RightRailBlock children={layoutAreas.map((name) => layoutItem(name))} />
 	</div>
 );
+
+export const commerce = () => (
+	<div id="fusion-app" className="layout-section">
+		<RightRailBlock children={layoutAreas.map((name) => layoutItem(name))} />
+	</div>
+);
+commerce.parameters = {
```

Comment:
@visionbegin We can set a story to use a given theme by changing it's parameters like so